README.md

```diff
@@ -1,6 +1,14 @@
 # LoopBack Push Notification Component
 
-![StrongLoop Labs](http://docs.strongloop.com/download/thumbnails/5310165/StrongLoop%20Labs%20Logo%20Cropped.png "StrongLoop Labs")
+**⚠️ LoopBack 3 is in Maintenance LTS mode, only critical bugs and critical
+security fixes will be provided.**
+
+We urge all LoopBack 3 users to migrate their applications to LoopBack 4 as
+soon as possible. Refer to our
+[Migration Guide](https://loopback.io/doc/en/lb4/migration-overview.html)
+for more information on how to upgrade.
+
```

Comment:
do we need to add the LTS policy for this module? e.g. https://github.com/strongloop/loopback-component-explorer#module-long-term-support-policy

Reply:
Done, LGTY now?